tests/unit/test_rds.py

```diff
@@ -125,6 +136,7 @@ def _db_run(self, option):
         connection = get_connection_dict(args)
         alchemy = DB_DICT[option](connection)
         alchemy.run_test(args.source)
+        assert check_if_string_in_file(LOG_PATH, LOG_STRING_COMMIT)
```

Comment:
As `unittest` is used, `self.Assert...` methods should be used instead of `assert`